RELEASE/scripts/autoscend/auto_quest_level_13.ash

```diff
@@ -496,23 +705,12 @@ boolean L13_towerNSHedge()
 
 boolean L13_sorceressDoor()
 {
-	if(internalQuestStatus("questL13Final") < 5 || internalQuestStatus("questL13Final") > 5)
+	if(internalQuestStatus("questL13Final") != 5)
 	{
 		return false;
 	}
 
-	if (item_amount($item[white pixel]) >= 30 && item_amount($item[Digital Key]) == 0)
-	{
-		if (in_koe())
-		{
-			buy($coinmaster[Cosmic Ray\'s Bazaar], 1, $item[Digital Key]);
-		}
-		else
-		{
-			cli_execute("make digital key");
-		}
-		set_property("auto_crackpotjar", "finished");
```

Comment:
I think this crackpot jar handling was missed

Reply:
looks like this was always wrong. It should've been "done" rather than "finished" (Cheesecookie seemed to make a lot of these types of mistakes). But this also looks like it's handled inside `LX_getDigitalKey()` now in @taltamir's changes.